lib/rspec/core/formatters/documentation_formatter.rb

```diff
@@ -6,12 +6,19 @@ module Core
     module Formatters
       # @private
       class DocumentationFormatter < BaseTextFormatter
-        Formatters.register self, :example_group_started, :example_group_finished,
+        Formatters.register self, :example_started, :example_group_started, :example_group_finished,
                             :example_passed, :example_pending, :example_failed
 
         def initialize(output)
           super
           @group_level = 0
+
+          @example_started = false
```

Comment:
Its not the name of the method, its the state of the example, I prefer running as it indicated the state now, not what it was.